### PR TITLE
Fixing minor typos in es.ts

### PR DESCRIPTION
**Pull request by amantewhiskey**

### CODE REVIEW

**Review by tukusejssirs — APPROVED** [review]

**Comment by tukusejssirs** [comment]
Thanks, @amantewhiskey! :pray: I believe it was me who caused these typos, I’m sorry. 😅 😉

**Comment by tukusejssirs** [comment]
@emagnier, what is this `1 workflow awaiting approval`? Is it the new check you created in #461? Can I run it or should I leave it for you? 🤔

**Review by emagnier — APPROVED** [review]
Thank you for these fixes @amantewhiskey!

@tukusejssirs, the first time a new contributor submits a PR, the GitHub Action workflow does not run automatically (for security reasons). But once approved for this contributor, the next time he opens a PR on this repository, it will now run automatically.
Here, I've approved it. Feel free to do the same when it appears for future new contributors.

**Pull request merged**
